### DISCUSSION TITLE
How to remove the Pokémon icons that are seen in the terminal?

### DISCUSSION THREAD

**Discussion by JonaBadd** [prasanthrangan/hyprdots#998] [Q&A]
good morning everyone!  I would like to know if Pokémon characters can be removed from the console (terminal) and how it can be done.  From already thank you very much!

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/issues/74

**Reply by JonaBadd** [reply]
Thank you
